src/parser/utils.ts

```diff
@@ -27,3 +27,23 @@ export function isMap(d: unknown): d is Map<unknown, unknown> {
 export function isDate(d: unknown): d is Date {
   return Object.prototype.toString.call(d) === '[object Date]';
 }
+
+/** @internal */
+export type StylizeFunction = (x: string, style: string) => string;
+export type InspectParameterFn = (x: unknown, options: unknown) => string;
+// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+export const basicInspectParameterFn: InspectParameterFn = v => `${v}`;
```

Comment:
```suggestion
export const defaultInspect: InspectParameterFn = v => `${v}`;
```
We're using this as a default, so maybe using "default" in the name provides more information than "basic"

Reply:
sounds good! I'll change it.